ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh

```diff
@@ -100,7 +100,7 @@ function getCIR(){
 #CLUSTERTYPE can be one of "virt", "virt-arm64" or "baremetal"
 CIRTYPE=host
 [ "$CLUSTERTYPE" == "baremetal" ] && CIRTYPE=cluster
-[ "$CLUSTERTYPE" == "virt-arm64" ] && CIRTYPE=host-arm
+[ "$CLUSTERTYPE" == "virt-arm64" ] && CIRTYPE=host_arm
```

Comment:
@derekhiggins did we maybe documented this one already?

Reply:
I've updated the maintanence doc to include all the correct pools we use